README.md

```diff
@@ -37,67 +21,220 @@ Hello
+    <li><a href="https://www.w3schools.com/css">CSS</a></li>
   </ul>
+  
 </details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+1 Added index.html file
+2 Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Fork this repoository to your desired folder
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Godedict/Hello-Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+Run this command command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  Open index.html using live server and view in your wb browser
+```
+
+<!-- AUTHORS -->
+
+## 👥 Author
+
+- Aderounmu Adeniyi
+Github: [@githubhandle] - (https://github.com/godedict)
+
+Likedin: [@linkedinhandle] - (https://www.linkedin.com/in/adeniyi-aderounmu-a03463111/)
+
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features.
+
+- [ ] Add home menu
+- [ ] Add media contents
+- [ ] Add footer
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🚀 Live Demo <a name="live-demo"></a>
+<!-- CONTRIBUTING -->
 
-> Add a link to your deployed project.
+## 🤝 Contributing <a name="contributing"></a>
 
-- [Live Demo Link](https://google.com)
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 💻 Getting Started <a name="getting-started"></a>
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project, kindly give me a star  rating.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse for this opportunity.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
 
-> Describe how a new developer could make use of your project.
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p
+# Hello-Microverse
+ 
+## About the Project
+Ths project is part of my exercise in the training
+
+ ## Built With
+ - HTML and CSS
```

Comment:
- [ ] I noticed that there is an old `README` below the corrected one. I assume this might have been an oversight, so would you kindly remove it to avoid confusion and ensure that your project looks more professional? 💯